src/Compilers/VisualBasic/Test/Emit/BasicCompilerEmitTest.vbproj

```diff
@@ -53,6 +54,18 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="Microsoft.VisualBasic" />
+    <PackageReference Include="Microsoft.DiaSymReader">
```

Comment:
Because in a new SDK project `PackageReference` can be a single line, but when applied to an existing desktop project it must be three lines.  😦